src/workflows/registration.ts

```diff
@@ -54,7 +42,7 @@ export async function registerOffchainWorkflow({
     },
   });
 
-  return;
+  return registeredUser.data;
```

Comment:
This will always return `{tx_hash: ""}` because that is what api returns for successful responses.
https://github.com/immutable/imx-engine/blob/257f041ca490c60846742e8d8e51e1e8dfda6c78/services/engine/src/controllers/userregistration/registration.go#L189

Reply:
That is what we discussed to return right?

Reply:
It was `void` before, returning the underlying api call's response makes more sense to me

Reply:
@Kastet Fair point. If any change should happen it should happen upstream.